docs/requesting-libraries.md

```diff
@@ -12,14 +12,18 @@ First check that the package is not already available, by [viewing all installed
 OpenSAFELY R image](https://github.com/opensafely-core/r-docker/blob/main/v2/packages.md). If your package isn't there then [open an
 issue](https://github.com/opensafely-core/r-docker/issues) to request it be added. Include the following information:
 
-* A link to the CRAN page for the package;
+* A link to the CRAN page or the (GitHub/GitLab etc.) repository for the package;
 * The specific package version you need, if any;
 * A brief explanation of why the package is needed.
 
 We consider the following things when deciding whether to include a new R package:
 
-* Is the package available on [CRAN](https://cran.r-project.org/)? We don't currently support packages that are not on CRAN but feel free to open an issue if you think it warrants further discussion.
+* The quality of the package. The vast majority of our packages come from CRAN. In r:v2 we can install from a remote repository, for such packages the package must build and be able to be added to our [r-universe](https://opensafely-core.r-universe.dev/).
 * Is the package "in scope"? In other words does the package help to process and analyse OpenSAFELY-type data, and is this clear from the request? If it's not clear what the package will be used for, we may ask for more information to justify its inclusion, so try to communicate this from the start. Out-of-scope includes:
+    * Is this a package you just require for example post processing released outputs. If so, in r:v2 you can install your own local packages simply by running
```

Comment:
@remlapmot I think we want to link the the documentation about this in r-docker repo?

Reply:
Good point - I have added a link to the r-docker README in dc2b1a8eb1f3744436f8ed4af15b8fcb96bc3466 .